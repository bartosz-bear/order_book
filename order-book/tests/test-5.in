{"type": "Iceberg", "order": {"direction": "Sell", "id": 1, "price": 14, "quantity": 20, "peak": 1}}