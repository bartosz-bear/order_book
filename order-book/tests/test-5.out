{"sellOrders": [{"id": 1, "price": 100, "quantity": 150}], "buyOrders": []}
{"buyOrderId": 1, "sellOrderId": 1, "price": 100,  "quantity": 100}
{"sellOrders": [{"id": 1, "price": 100, "quantity": 50}], "buyOrders": []}
